app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @mechanic.name %></h1>
+<h3>Years of Experience: <%= @mechanic.years_experience %></h3>
+<h3>Rides: <%= @mechanic.rides.pluck(:name).join(", ") %></h3><br>
+<h2>Add a ride to workload:</h2>
+<%= form_with url: "/mechanic_rides/", method: :post, data: { turbo: false } do |form| %>
```

Comment:
Excellent implementation of this form. Another RESTful option in case you don't want to have to pass a hidden field for mechanic_id is to use this path: `/mechanics/mechanic_id/rides`